doozer/doozerlib/cli/images_konflux.py

```
@@ -535,16 +550,27 @@ async def run(self):
 
         results = await asyncio.gather(*tasks, return_exceptions=True)
         failed_tasks = []
+        successful_tasks = []
         for dgk, result in zip(dgk_records, results):
```

Ruff (0.15.0)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)